_docs/_latest/setup/install.md

```diff
@@ -79,7 +105,7 @@ Create a file named `main.tf` in an empty directory and copy the contents below
 ```hcl
 module "forseti" {
   source  = "terraform-google-modules/forseti/google"
-  version = "~> 5.0.0"
+  version = "~> 5.2.0"
```

Comment:
5.2 is not out yet, should this be part of the 2.25 release? Omit the micro version from this syntax, otherwise we will need to update this version for every release. This can be `~> 5.0` or `~> 5.1` which will use the latest release up until 6.x.

Reply:
Updated version to `5.1`.